### ISSUE TITLE
Synching option for Google Drive or Dropbox

### ISSUE THREAD

**Issue by tazihad** [xbrowsersync/app#277]
There's many server in the website. But It's not trusting enough for me to sync bookmarks to their servers. What happens if they decided to stop their hosting?
Can you make a backup syncing option for Google Drive or Dropbox?
There's a similar project named [syncmarx](https://github.com/Cleod9/syncmarx-webext) which has implemented this. But the project is abandoned I guess.

**Comment by da2x** [comment]
(It's already on the roadmap.)[https://github.com/xbrowsersync/app/wiki/Roadmap]

**Comment by nero120** [comment]
@tazihad as @da2x says support for syncing to other cloud hosts is planned. Rename this feature request for either Google Drive or Dropbox (and if you want create another request for the other) and that way we can track demand for the specific host.

**Comment by candrapersada** [comment]
if I sync using Google Drive or Dropbox, do I still have to use Sync ID or account?

**Comment by nonoroazoro** [comment]
> if I sync using Google Drive or Dropbox, do I still have to use a Sync ID or use an account?

I'd say yes.

**Comment by nero120** [comment]
> if I sync using Google Drive or Dropbox, do I still have to use a Sync ID or use an account?

@candrapersada yes you would have a sync ID (you might want to have multiple syncs stored in your cloud storage account), but rememver there are no "accounts" in xBrowserSync!

**Comment by candrapersada** [comment]
I mean login using Google Drive or Dropbox, or login using Sync ID to sync bookmarks?

**Comment by nero120** [comment]
Your google drive/dropbox is account protected, so you would definitely have to log in before xBrowserSync could access it. Once authorised, you'd then create a new sync with xBrowserSync like normal and you'd get a sync ID, which you can then use to re-sync that data.

**Comment by cjpalermo21** [comment]
Was this resolved? Apparently, I *exceed* the data limit; so I cannot use XBrowserSync; but, obviously, if I can use Google Drive as my "server" - that won't be an issue. :/
